### ISSUE TITLE
Reg\Dashboard - Internal Users can see dashboard tiles

### ISSUE THREAD

**Issue by shon-button** [bcgov/cas-registration#2149]
#### Description:

As an internal user, I have a registration dashboard, so that I can view operators' information, events, and make approvals
[Internal user Figma link](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7781-6399&mode=design&t=KFDAnKMzwJ3b1l14-4)

For the styling (font, colour, margin, padding, etc.), please refer to [this frame on the BCIERS Design Guidelines page](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?node-id=11103-107&t=pmKjK1RmQeLYOb04-4) in Figma.

#### Acceptance Criteria:
Given I'm an IRC user and I have access to the Registration tile on the BCIERS dashboard
When I click the Registration link in the Registration tile
Then I see the Internal - Registration page as in the [design](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14644-2&t=pU2kztfH9zOcYdIY-4)
When I click on the sub links of each tile
Then I'm navigated to the proper page as in the [prototype](https://www.figma.com/proto/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?page-id=7781%3A6398&type=design&node-id=7781-6399&viewport=74%2C199%2C0.46&t=xuYjeIKfjwjoCJNb-1&scaling=scale-down&starting-point-node-id=7781%3A6471)

#### Development Checklist:
- [ ] in `bciers/apps/registration/app/dashboard/page.tsx` get tile JSON data using `fetchDashboardData from "@bciers/actions/server";` with parameter `"common/dashboard-data?dashboard=registration"`
- [ ] send the tile JSON data to `Tiles from "@bciers/components/navigation/Tiles";`
- [x] make Tiles pretty

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #1617

**Comment by shon-button** [comment]
@patriciarussellCAS

Reachable at:
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/registration
